### DISCUSSION TITLE
MIT 6.042J: Mathematics for Computer Science - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#136] [Announcements]
# MIT 6.042J: Mathematics for Computer Science - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%95%B0%E5%AD%A6%E8%BF%9B%E9%98%B6/6.042J/

**Comment by YKHG** [comment]
calculus基础 必学吗

**Reply by taylover2016** [reply]
Calculus 在这门课当中用到的还挺多，不过集中在积分以及级数这两块。推荐还是先了解相关的知识再来学这门课。

**Comment by AAAAyl0n** [comment]
有作业答案吗

**Reply by taylover2016** [reply]
官方没有给出答案，可以上GitHub搜一下看看有没有；我自己偷懒没做(^_-)

**Reply by mikamijpen** [reply]
请问找到了吗

**Reply by Coodist** [reply]
我有，等我贴一个网盘链接

**Reply by Coodist** [reply]
链接：https://pan.baidu.com/s/1_dPnIC58znvCrqB4ryfRew 
提取码：1111

**Reply by Coodist** [reply]
05和10年的都在里面了，就是忘记是在哪里找的了

**Comment by paraliine** [comment]
请问这本教材的答案有吗？这上面的习题我都想做做试试。

**Reply by taylover2016** [reply]
我这边没有做题目😂不过上面的评论里面应该有的

**Reply by Elexs126** [reply]
等会儿，这有教材吗？说的是网站上的reading吗？

**Comment by M1s4kuy4** [comment]
😭这门课里面的数论真的不需要前置课程嘛？我本科数三的，微积分和线代没接触过模，完全看不懂…但不知道找什么前置课程去看

**Reply by taylover2016** [reply]
数论部分看不懂正常哈，本身这部分就很难的。如果不想看前置课程的话，直接记住有关 multiplicative inverse 的内容就好（不过我个人感觉看了前置课程还是会看不懂lol

**Reply by M1s4kuy4** [reply]
哈哈哈哈哈真的看不懂mod运算和乘法逆元，我已经去B站看初中竞赛数论的基础视频了，先把整数基础了解一下再继续死磕6.042

**Comment by codeonthepath** [comment]
这门课和cs70，看哪个比较好啊

**Reply by xiechen274** [reply]
同问

**Comment by M1s4kuy4** [comment]
Cs70我没有看过。6.042J我看到分治递归就没有继续看了。如果你距离求职还有2年左右的时间的话我觉得离散数学看一看应该会有所帮助。

 

发件人: ***@***.*** ***@***.***> 代表 meow
发送时间: 2023年6月7日 21:24
收件人: PKUFlyingPig/cs-self-learning ***@***.***>
抄送: M1s4kuy4 ***@***.***>; Comment ***@***.***>
主题: Re: [PKUFlyingPig/cs-self-learning] MIT 6.042J: Mathematics for Computer Science - CS自学指南 (Discussion #136)

 

这门课和cs70，看哪个比较好啊

**Comment by AlexanderWang888** [comment]
有一起的吗，qq：2581606394

**Comment by zhangbird050801** [comment]
图论部分最推荐哪个视频啊

**Reply by shiquda** [reply]
推荐一个最近发现的视频系列，当作初步了解挺不错的，风格很像3B1B

https://www.youtube.com/watch?v=oXcCAAEDte0&list=PLn9qm_iibFqwjD1TUL8Rvg9rKN7-H8ZNy

**Comment by dghtucs** [comment]
建了一个课程QQ群，想讨论可入611032363

**Comment by RisingUppercut** [comment]
2024 spring 的6.1200（6.042的新课号）的视频已经能看到了，我无意中刷到的，应该是接下来ocw要开放的课程：   https://archive.org/details/mit6_1200js24

**Reply by NEU-china-top-fei** [reply]
这个24版的作业有答案吗xd

**Reply by minwang-ai** [reply]
大家都是怎么知道最新的课的呀？

**Comment by Elexs126** [comment]
24 spring 可以直接在b站上看,搜6.1200即可

**Comment by yinbowen-dc** [comment]
这个是我使用AI 自动进行语音识别的中文翻译视频，翻译后使用deepseek做过优化，应该能帮助大家少走一些弯路
https://www.bilibili.com/video/BV1RBGCzbE9J/?spm_id_from=333.1387.homepage.video_card.click&vd_source=843c61b5b2e08729a01e31914d159fc4
我是darkchunk 全平台同名

**Comment by RisingUppercut** [comment]
2024sp的资源已全部开放：

https://ocw.mit.edu/courses/6-1200j-mathematics-for-computer-science-spring-2024/lists/problem-sets/

**Comment by lil-Journey** [comment]
自建日本节点服务器 月租5元不限流量 本人也在用 失效时可联系人工修复 qq:1165116287

**Comment by mloluyu** [comment]
请问MIT6.1200J和RES6.012的学习路线怎么样

**Comment by kawayee** [comment]
mit前几年改课号了，现在叫6.1200，推荐看2024-Erik Demaine（就是那个疑似地中海的教授）这一版，实际听下来+结合gemini的评价，2024版结合了非常多的程序化思维。2010版课堂上的内容并不能覆盖全书，有一部分放在recitation里了，2010版Tom教授讲的很有启发性，另一个老师Marten讲的跳跃性非常大，边界条件给的也不完善。
综合下来，gemini虽然出现了幻觉（RSA的主讲不是Tom），但给的意见还是可以听的：以2024为主线，听不懂的地方去看2010-tom讲的部分，2010-Marten直接跳过。

**Reply by ddddddavid66** [reply]
CS70比起来你觉得哪个更好啊
